frontend/functions/data_functions.py

```
@@ -386,3 +387,152 @@ def build_mountain_map(df_json: dict, min_height: int, counties: list) -> folium
         ).add_to(m)
 
     return m
+
+
+def create_route_animation(ids: list[str], vars: Variables) -> folium.Map:
```

Ruff (0.15.6)

[error] Function argument `vars` is shadowing a Python builtin

(A002)